icommons_ext_tools/settings/base.py

```diff
@@ -238,6 +238,8 @@
 
 COURSE_WIZARD = {
     'OLD_LMS_URL': SECURE_SETTINGS.get('old_lms_url'),
+    'TERM_TOOL_BASE_URL' : 'https://isites.harvard.edu',
```

Comment:
We should actually pull this in from secure.py, then you don't need to customize in the other environment specific settings files.